examples/absent-record.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$Version": "4.01",
+    "foo": {
+        "bar": {
+            "$Kind": "Term",
+            "$Type": "foo.record",
+            "$Nullable": true
+        },
+        "record": {
+            "$Kind": "ComplexType",
+            "@foo.bar#WithRecord": {},
+            "@foo.bar#WithoutRecord": true
```

Comment:
The default annotation value is JSON depends on whether a `<Record>` was present in XML.

Reply:
Correct. We use Boolean tagging annotations (typically with `Type="Core.Tag" DefaultValue="true"`) a lot, and the code is optimized for this.

This can only be improved if the term definition is read from the referenced vocabulary, which is currently not the case.